src/main/java/me/karakelley/tictactoe/PlayerTypeValidator.java

```diff
@@ -0,0 +1,16 @@
+package me.karakelley.tictactoe;
+
+public class PlayerTypeValidator implements Validator {
+  @Override
+  public boolean isValidInput(String input) {
+    try {
+      return inBounds(Integer.parseInt(input));
+    } catch(IllegalArgumentException e) {
+      return false;
+    }
+  }
+
+  private boolean inBounds(int choice) {
+    return choice > 0 && choice < 5;
```

Comment:
The logic for "what a valid choice is" is duplicated between this class and the factory. Can you think of a way to refactor?